src/App.css

```diff
@@ -279,11 +279,26 @@ aside{
 .css-1t8x7v1 >.ps-menu-button{
   color: #fff!important;
   padding-left: 40px!important;
-  /* padding-right: 0px!important; */
-
+}
+.bg-view{
+  background-color: rgb(14,56,47)!important;
 
- 
 }
+.view input{
+  background-color: #EF9B28!important;
+}
+.arrView{
+height: 30px!important;
```

Comment:
I recommend you to use "rem" instead of "px", and you can read about the different between the rem and px